roles/etcd_ca/tasks/main.yml

```diff
@@ -38,9 +38,9 @@
   delegate_to: "{{ etcd_ca_host }}"
   run_once: true
 
-- command: touch {{ etcd_ca_db }}
-  args:
-    creates: "{{ etcd_ca_db }}"
+- file:
+    path: "{{ etcd_ca_db }}"
+    state: touch
```

Comment:
I think the real point that @abutcher is trying to make is that we
shouldn't me modifying something every time (read: idempotentcy) if the
modification does not need to happen. An ideal playbook run on an existing
cluster should say **0 changes**.

On Fri, Sep 2, 2016 at 10:27 AM, Manjunath A Kumatagi <
notifications@github.com> wrote:

> In roles/etcd_ca/tasks/main.yml
> <https://github.com/openshift/openshift-ansible/pull/2398#discussion_r77382707>
> :
>
> > @@ -38,9 +38,9 @@
> >    delegate_to: "{{ etcd_ca_host }}"
> >    run_once: true
> >
> > -- command: touch {{ etcd_ca_db }}
> > -  args:
> > -    creates: "{{ etcd_ca_db }}"
> > +- file:
> > +    path: "{{ etcd_ca_db }}"
> > +    state: touch
>
> Touching file for subsequent run will not make any changes though except
> the last modification date for the file. And the warning I'm trying to
> avoid in this PR basically [WARNING]: Consider using file module with
> state=touch rather than running touch
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/openshift-ansible/pull/2398/files/d40e07a3291f736c397abcf9d0aaca61b2a5d833#r77382707>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AACBYUU62XF75UZDLHCT7tl6uUDEGYjNks5qmFyKgaJpZM4Jzq_p>
> .
>



-- 
Tim Bielawa, Software Engineer [ED-C137]
Cell: 919.332.6411  | IRC: tbielawa (#openshift)
1BA0 4FAB 4C13 FBA0 A036  4958 AD05 E75E 0333 AE37